components/auth/Login.tsx

```diff
@@ -140,13 +156,38 @@ const Login: FC = () => {
         });
         Router.push('/issuer');
       }
+      // -----------user------------ //
+
+    } else if( mode === 'user' ) {
+      const [err, user] = await signIn({
+        userInfo: {
+          email: account,
+          password,
+        },
+      });
+      if (!user || user.length === 0) {
+        notify.error({msg: err || '此帳號不存在' });
+      } else if (!user[0].isIssuer) {
+        notify.error({ msg: '此帳號不是使用者帳號，請確認使用者帳號'})
+      } else {
+        updateUser({
+          ...preparedUser(user[0]),
+          loginMode: 'user',
+        });
+        // useContext update(Change) User
+        Router.push('/pages/issuer/product');
+      }
+      
+      // -----------user------------ //
     } else if (validate()) {
```

Comment:
Seems the user login flow is solved by  https://github.com/Turing-Chain/certs.turingchain.tech/pull/4
Please fetch the newest commit and move on your pr.
---
hint: I think you just need to add the onRegister func with UI Button in this page.

Reply:
fix